packages/common/log/src/config.ts

```diff
@@ -37,6 +37,7 @@ export enum LogProcessorType {
   CONSOLE = 'console',
   BROWSER = 'browser',
   DEBUG = 'debug',
+  TIMESTAMPED = 'timestamped',
```

Comment:
how this one is used?

Reply:
disabled by default, see example usage in https://github.com/dxos/dxos/pull/4219/files#diff-caf5ec8794e5abcf732f93c24f7505bb212274e3dc1ae51562a78aa6c8927fe3